meganav.yaml

```diff
@@ -0,0 +1,951 @@
+            - title: Ubuntu code of conduct
+              description: Expectations of participants
+              url: /community/code-of-conduct
+            - title: Diversity policy
+              description: Our stance on diversity
+              url: /community/diversity
+            - title: Project governance
+              description: How the project is governed
+              url: /community/governance
+            - title: Community Council
+              description: Governing body for the project
+              url: https://discourse.ubuntu.com/t/what-is-the-ubuntu-community-council/706
+            - title: Teams
+              description: The teams in the project
+              url: /community/governance/teams
+            - title: Local community teams
+              description: Join a community team
+              url: http://loco.ubuntu.com/teams/
+
+      secondary_links:
+        - title: Quick links
+          links:
+            - title: Community Council agenda
+              url: https://discourse.ubuntu.com/t/community-council-meeting-agendas-schedule/708
+            - title: Ubuntu Technical Board
+              url: https://wiki.ubuntu.com/TechnicalBoard
+            - title: Technical Board agenda
+              url: https://wiki.ubuntu.com/TechnicalBoardAgenda
+
+    - title: Keep up to date
+      primary_links:
+        - links:
+            - title: Read the blogs
+              description: Ubuntu blogs
+              url: /blog
+            - title: Weekly Newsletter
+              description: Sign up for our newsletter
+              url: https://wiki.ubuntu.com/UbuntuWeeklyNewsletter
+            - title: Mailing lists
+              description: All the public mailing lists
+              url: https://lists.ubuntu.com/
+            - title: Ubuntu Planet
+              description: Blogs by Ubuntu members
+              url: http://planet.ubuntu.com/
+            - title: Teams
+              description: The teams in the project
+              url: https://wiki.ubuntu.com/Teams
+
+      secondary_links:
+        - title: Quick links
+          links:
+            - title: Become a blog contributor
+              url: https://community.ubuntu.com/t/call-for-contributors-to-the-ubuntu-weekly-newsletter/1542
+            - title: Facebook
+              url: https://www.facebook.com/ubuntulinux
+            - title: Twitter
+              url: https://twitter.com/ubuntu
+            - title: YouTube
+              url: https://www.youtube.com/c/UbuntuOS
+            - title: GitHub
+              url: https://github.com/ubuntu
+
+get_ubuntu:
+  side_nav_sections:
+    - title: Desktop
+      primary_links:
+        - links:
+            - title: Ubuntu Desktop
+              description: Free and reliable desktop OS
+              url: /download/desktop
+              secondary_cta_title: Download 22.04 LTS
+              secondary_cta_url: /download/desktop/thank-you?version=22.04.3&architecture=amd64
```

Comment:
Shouldn't these values come from the releases.yaml?

Reply:
Yes, good point. I have made an issue in our backlog to address this later: https://warthogs.atlassian.net/browse/WD-8596